docs/pages/specs/roadmap.mdx

```diff
@@ -15,12 +15,12 @@ The Nexus zkVM and Nexus Network are both undergoing rapid development.
 
 Contribute or follow along with the latest updates to the system on [Github](https://github.com/nexus-xyz/nexus-zkvm).
 
-### v1.0 to v2.0 to Beyond
```

Comment:
I would change this to `3.0 to Beyond` -> these changes are better traced by the marketing label than the software version one.

Reply:
So it will be talking about "3.0 to Beyond" and "Nexus 2.0" system. I'll remove `v`s in front of these.

Reply:
done.